.github/workflows/documentation.yaml

```diff
@@ -13,9 +13,10 @@ jobs:
           pip install sphinx
           python -m pip install sphinx-autoapi
           python -m pip install sphinx_rtd_theme
+          python -m pip install tox tox-gh-actions
       - name: Sphinx build
         run: |
-          sphinx-build doc _build
+          tox -e docs
       - name: Deploy
         uses: peaceiris/actions-gh-pages@f09b4d7e2f12f433806a6273c6b0025df1f7859d
```

Comment:
Also, would update this to `uses: peaceiris/actions-gh-pages@v3.9.3` instead of commit id.